.github/workflows/pr_push_workflow.yml

```diff
@@ -0,0 +1,58 @@
+name: PR/push workflow
+
+on: [push, pull_request]
+
+jobs:
+    Fast_build:
+        # uses: oneapi-src/unified-memory-framework/.github/workflows/fast_build.yml@main
```

Comment:
why this line is commented? I thought you want to call fast_build here

Reply:
For now, I remove this comment and related file.